thetis/turbines.py

```diff
@@ -42,19 +43,23 @@ def power(self, uv, depth):
         alpha = self.velocity_correction(uv, depth)
         A_T = pi * self.diameter**2 / 4
         uv3 = dot(uv, uv)**1.5 / alpha**3  # upwind cubed velocity
-        C_T = self.thrust_coefficient(uv3**(1/3))
+        C_P = self.power_coefficient(uv3**(1/3))
         # this assumes the velocity through the turbine does not change due to the support (is this correct?)
-        return 0.25*C_T*A_T*(1+sqrt(1-C_T))*uv3
+        return 0.5*(physical_constants['rho0']/1000)*A_T*C_P*uv3
```

Comment:
Yes, I had kept it consistent with previously in case there was a reason for this. I agree with changing it to Watts. Inclusion of the density is necessary as most thrust and power curves expect it to be 1025-1026kg/m3, not 1000kg/m3.